Code/Legacy/CrySystem/Timer.h

```diff
@@ -21,7 +21,7 @@ class CTimer
     // constructor
     CTimer();
     // destructor
-    ~CTimer() {};
+    ~CTimer() = default;;
```

Comment:
1. extra semi-colon  ';;'
2. Shouldnt this be `~CTimer() override = default;  `

Reply:
I used a tool to do those fixes and because there was a previous `;` it left it there. Removing